project/src/main/world/goop-ground-map.gd

```diff
@@ -0,0 +1,28 @@
+tool
+extends TileMap
```

Comment:
Add class level comment.

Reply:
Added comment: `Updates the goop ground map shader with its view_to_local shader parameter.`